packages/portal-proto/src/features/cart/updateCart.tsx

```diff
@@ -64,9 +65,11 @@ const AddNotification: React.FC<AddNotificationProps> = ({
     currentCart.map((c) => c.file_id).includes(f.file_id),
   );
 
-  if (filesToAdd.length > 0) {
-    dispatch(addFilesToCart(filesToAdd));
-  }
+  useEffect(() => {
```

Comment:
The bug fix, caused by this: https://dev.to/console_logs/warning-cannot-update-a-component-while-rendering-a-different-component-5bfk